tools/run_tests/run_tests.py

```diff
@@ -332,6 +332,8 @@ def _compiler_options(self, use_docker, compiler):
       return ('wheezy', self._gcc_make_options(version_suffix='-4.4'))
     elif compiler == 'gcc4.6':
       return ('wheezy', self._gcc_make_options(version_suffix='-4.6'))
+    elif compiler == 'gcc4.8':
+      return ('wheezy', self._gcc_make_options(version_suffix='-4.8'))
```

Comment:
I'm wondering if wheezy is the right choice here or if this should be based on jessie instead, in the overall spirit of choosing newer platforms for C++. In either case, I think you'll need to apt-get gcc-4.8/etc in the Dockerfile template.